src/util.ts

```diff
@@ -0,0 +1,16 @@
+function gen4() {
+  return Math.random().toString(16).slice(-4);
+}
+
+export function simpleUniqueId(prefix: string = "") {
+  return `${prefix}${[
+    gen4(),
```

Comment:
Do we need that much entropy in an ID?

Reply:
I suppose there's a pretty low chance for collision and an actual collision wouldn't affect much in terms of functionality. I'm just being overly cautious probably.